person.rb

```diff
@@ -0,0 +1,25 @@
+# Person Class
+class Person
+  def initialize(age,
+                 name = 'Unknown',
+                 _parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
```

Comment:
- [ ] I would recommend you move this declaration of setters and getters to the top of the class for better code organization.